core/shared/platform/windows/platform_internal.h

```diff
@@ -137,9 +140,47 @@ bh_atomic_thread_fence(int mem_order);
 
 #define os_atomic_thread_fence bh_atomic_thread_fence
 
-typedef HANDLE os_handle;
-typedef void os_dir_stream;
+typedef enum windows_handle_type {
+    windows_handle_type_socket,
+    windows_handle_type_normal,
+    windows_handle_type_stdio
```

Comment:
I didn't see the use of those two in the PR - is that still in progress? Also, not sure what `normal` means - is it for files? If so, maybe we can be specific.

Reply:
will change to `windows_handle_type_file`